.github/workflows/ci/pkg/bot/check.go

```diff
@@ -29,8 +29,14 @@ import (
 
 // Check checks if all the reviewers have approved the pull request in the current context.
 func (c *Bot) Check(ctx context.Context) error {
-	pr := c.Environment.Metadata
-	if c.Environment.IsInternal(pr.Author) {
+	author := c.Environment.Metadata.Author
+	// Assign reviewers again, as the required reviewers may have changed since
+	// the initial assign workflow (due to new changes, for example)
+	err := c.Assign(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
Can you check if we can update the events that assign gets triggered on and just have assign re-run when changes are made?

Reply:
See PR description.